code-challenges.test.js

```diff
@@ -28,42 +33,172 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
+
+
+// describe("capitalizedArray", () => {
+//     it("returns an array with all the words capitalized", () => {
+//         expect(randomNouns1.map.toUpperCase).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+//         expect(randomNouns2.map.toUpperCase).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+
+//     })
+// })
+
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+
+
+// FAIL  week-2-assessment-VMMarin/code-challenges.test.js
+// capitalizedArray
+//   ✕ returns an array with all the words capitalized (2 ms)
+
+// ● capitalizedArray › returns an array with all the words capitalized
+
+//   expect(received).toEqual(expected) // deep equality
+
+//   Expected: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+//   Received: undefined
+
+//      98 | describe("capitalizedArray", () => {
+//      99 |     it("returns an array with all the words capitalized", () => {
+//   > 100 |         expect(randomNouns1.map.toUpperCase).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+//         |                                              ^
+//     101 |         expect(randomNouns2.map.toUpperCase).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+//     102 |
+//     103 |     })
+
+//     at Object.<anonymous> (week-2-assessment-VMMarin/code-challenges.test.js:100:46)
+
+
+
+// b) Create the function that makes the test pass.
+
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+//Pseudocode
+// Create a function that takes in an array of words and returns an array with all the words capitalized.
+// create a function that takes in an array and returns each word capitalized
+// declare a function that takes in an array
+// create a local variable that maps through the array
+// with each value access the first index and capitalize the letter then concatenate the remainder of the word
+// return the local variable joined into a string
 
-// b) Create the function that makes the test pass.
+const capitalNouns = (array) => {
+  let eachItem = array.map(value => {
+    return value[0].toUpperCase() + value.substring(1)
+  })
+   return eachItem.join(", ")
+}
 
+console.log(capitalNouns(randomNouns1))
+console.log(capitalNouns(randomNouns2))
```

Comment:
Your logic looks great. The test should call your function so you don't need lines 162/163.